CONTRIBUTING.md

```diff
@@ -60,7 +60,7 @@ Finally, the MONAI tutorial has a [README file](README.md) to communicate the im
 All source code and notebook files should include copyright information at the top of the file.
 
 NOTE: for Jupyter Notebook `.ipynb` files, the copyright information should appear at the top of the first markdown cell.
-There are extra two spaces at the end of each line to ensure no line auto-wrap in the markdown rendering in the display.
+There are *two extra spaces* at the end of each line to ensure no line auto-wrap in the markdown rendering in the display, e.g. the first line of the copyright should be `"Copyright (c) MONAI Consortium  \n"` instead of `"Copyright (c) MONAI Consortium\n"` in the raw string format.
```

Comment:
have you tried copying the licence info into a jupyter notebook cell? it seems pasting it to a markdown cell will automatically trim the two extra spaces copied from here...

Reply:
That's a good point. CC @mingxin-zheng

Reply:
If the contributor copies from the [example notebook](https://github.com/Project-MONAI/tutorials/blob/main/.github/contributing_templates/notebook/example_feature.ipynb) and pastes down in the notebooks, that should be fine (at least in VS Code).

But the copyright text on contributing guidelines does not contain the extra spaces, because the pre-commit hook will remove it automatically, as in the commit 1bf1523. I think contributors may be confused.

Reply:
thanks for the update, now the copy pasting works fine